website/docs/docs/quickstarts/dbt-core/codespace-qs.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Quickstart for dbt Core using GitHub Codespaces"
+id: codespace
+description: "Quickly get set up to start running dbt commands in a codespace environment."
+sidebar_label: "GitHub Codespace quickstart"
+---
+
+In this quickstart guide, you’ll learn how to create a codespace and be able to execute the `dbt build` command from it in _less than 5 minutes_. 
+
+dbt Labs provides a [GitHub Codespace](https://docs.github.com/en/codespaces/overview) template that you (and anyone else) can reuse to create a complete dbt environment with a working and runnable project. When you create the codespace, the [dev container](https://docs.github.com/en/codespaces/setting-up-your-project-for-codespaces/adding-a-dev-container-configuration/introduction-to-dev-containers) creates a fully functioning dbt environment, connects to a DuckDB database, and loads a year of fictional Jaffle Shop data into the database for you. Jaffle Shop is a fictional cafe selling food and beverages in several US cities.
+
+:::tip Check out our Jaffle Shop template 
+The [README](https://github.com/dbt-labs/jaffle-shop-template#readme) for the Jaffle Shop template also provides instructions on how to do this, along with animated GIFs. You can check those out, too.  
+
+:::
+
+## Prerequisites
+
+- To use the dbt CLI, it's important that you know some basics of the terminal. In particular, you should understand `cd`, `ls` , and `pwd` to navigate through the directory structure of your computer easily.
+- You have a [GitHub account](https://github.com/join).
+
+## Create a codespace
+
+1. Go to the `jaffle-shop-template` [repository](https://github.com/dbt-labs/jaffle-shop-template) after you log in to your GitHub account. 
+1. Click **Use this template** at the top of the page and choose **Create new repository**.
+1. Click **Create repository from template** when you’re done setting the options for your new repository.
+1. Click **Code** (at the top of the new repository’s page). Under the **Codespaces** tab,  choose **Create codespace on main**. Depending on how you've configured your computer's setting, this either opens a new browser tab with VSCode running in it or opens a new VSCode window with the codespace in it. 
```

Comment:
Adding this for clarity since running these steps doesn't really produce any apparent VSCode branding on the screen expect for a brief line that goes away automatically and is easily missed
```suggestion
1. Click **Code** (at the top of the new repository’s page). Under the **Codespaces** tab,  choose **Create codespace on main**. Depending on how you've configured your computer's setting, this either opens a new browser tab with the Codespace IDE (VSCode) running in it or opens a new VSCode window with the codespace in it. 
```

Reply:
Will this only work with VSCode if they open it via app or will it work with any (most?) code editors?

Reply:
looks like GH Codespace supports other editors  https://github.com/features/codespaces depending on how the dev container is configured